data/graphire4-6x8.tablet

```diff
@@ -0,0 +1,33 @@
+# Wacom
+# Graphire4 6x8
+# CTE-440
```

Comment:
This needs to be updated to match the model number of your tablet (CTE-640, I believe?)